02_Util/test/certificate/client/Acme.test.ts

```diff
@@ -0,0 +1,121 @@
+import { readFileSync } from 'fs';
+import { SystemConfig } from '@citrineos/base';
+import { ILogObj, Logger } from 'tslog';
+import { faker } from '@faker-js/faker';
+import { Acme } from '../../../src/certificate/client/acme';
+import { aValidSignedCertificate } from '../../providers/ACME';
+import * as CertificateUtil from '../../../src/certificate/CertificateUtil';
+import { Client } from 'acme-client';
+
+jest.mock('fs', () => ({
+  readFileSync: jest.fn(),
+}));
+jest.mock('../../../src/certificate/CertificateUtil');
+
+describe('ACME', () => {
+  const mockTlsCertificateChain = faker.lorem.word();
+  const mockMtlsCertificateAuthorityKey = faker.lorem.word();
+  let mockCertUtil: jest.Mocked<typeof CertificateUtil>;
+  let mockClient: jest.Mocked<Client>;
+
+  let systemConfig: SystemConfig;
+  let logger: Logger<ILogObj>;
+  let acme: Acme;
+
+  beforeAll(() => {
+    global.fetch = jest.fn();
+    (readFileSync as jest.Mock)
+      .mockReturnValueOnce(mockTlsCertificateChain)
+      .mockReturnValueOnce(mockMtlsCertificateAuthorityKey)
+      .mockReturnValueOnce(faker.lorem.word());
+    mockCertUtil = CertificateUtil as jest.Mocked<typeof CertificateUtil>;
+
+    systemConfig = {
+      util: {
+        networkConnection: {
+          websocketServers: [
+            {
+              id: '3',
+              securityProfile: 3,
+              tlsCertificateChainFilePath: faker.lorem.word(),
+              mtlsCertificateAuthorityKeyFilePath: faker.lorem.word(),
+            },
+          ],
+        },
+        certificateAuthority: {
+          chargingStationCA: {
+            name: 'acme',
+            acme: {
+              env: 'staging',
+              accountKeyFilePath: faker.lorem.word(),
+            },
+          },
+        },
+      },
+    } as any;
+    logger = new Logger<ILogObj>({ name: expect.getState().currentTestName });
```

Comment:
I think you could leave out the logger even. the Acme class instantiates it as it's optional.